modules/ssh-keys.nix

```diff
@@ -1,144 +1,96 @@
 {
-  artemohanjanyan = [
-    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQMCTlz9c84peYN7xXp1NWmJP0mCYLxnw8llWw9FS1EQsdJZeTmJesX1It8dTQ4khAxCtBpAacWphgZjjQquz9KmnSp245Dft4JYVCoHRrwez9DCNn4RwkfmnJTYLHt9vB77yMSwluoKFD54KjTBaLlgdQ9XPdFt7xWkpdYPWbTmuEx7bEpLMgwnHVrZ24i+FxEaIbnkJXVRzMB4MKdoCzJauqGUHtvXDVRdxU1kw93x0i9fxr2XjZK76I98z8Uz2WiMBZwaXsFZ8XYiSNOomwsIjHCkH+657D11U0A62euVffogr8XM9q51zDyvhREcWFtdkKKRSSlQIB8uHqicW7"
-    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDdJlxUtDGFoEEtsXKxl7aR2CKso+rVNDK2oPqlENJgvspJpAdJ9yW3GpD3yDT0YK2klhitk4jH9bov1Xg76nbw3/VvF63M1s+PWlhZyzeQ+FkOwnJD7+9/UuxrK7W0OYtcUiG+/2kbpyMBcyTXvYhd7pXH0O5A3UXRCmcpFBZw0+lZ+NoY9NdRVctmJCYzXl8ZVr/gL3sghrudBWE0gvnYggRmAyPzVW2+dV7VcUw/IaDba1AaA9VI/FXiYPLCo7jFg1Yu7wEGh7K0dNI4YbelqsDayVVauJwhPBr3YpEcU1ASIVUegVU93GFT35Sv4VPNSA8xh8lTzozpCtdoj0pl"
-  ];
-  awson = [
-    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCwV8tam8peOJ50gnbYeRhT6PBC9tFXYdUWG0pkVBcCBp4KpP+W5MOzsYC7Q+9NJpn61Fg9rT+6CQpk+HTjlgeFu8IadJ55K/ToAd2mXFNRDk/lHMfvK8+epg0Wwnf8oQkIB8S/DfzQ9S2jlkBQhSpeRYk4KkhvZgDKQDrW0lzicJI0kVHnTm3s1yvhDB6jDXMY6TGUtk2L82pryqxiOpYOeOydPGrBlcBjxkZY4iQIjoqnNJxsH/cwl0W9JszglR90gVXH6q1qirqHeDmKwQoB1FILmblf2GNh+MwJNJM6J64s8DgDL9ChkEDGfuJ5UFZZrB114p1vTqlW2fB0zsMt"
-  ];
-  ayrat555 = [
```

Comment:
I'm not sure about Ayrat, because he still helps us with Pont sometimes. So I can imagine a situation when something goes wrong with Pont and Ayrat needs to view logs or something. Usually we can share logs with him, but maybe he'll need to view them in real-time.

Anyway, it doesn't seem very likely and he doesn't need SSH access to anything else, so I think it's fine to remove him. In this unlikely scenario we will temporarily grant him access.

Reply:
> So I can imagine a situation when something goes wrong with Pont and Ayrat needs to view logs or something

AFAICS, Ayrat didn't have sufficient access to servers where pont is deployed